e2e/snapshot_ops_test.go

```diff
@@ -0,0 +1,239 @@
+package e2e
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gluster/glusterd2/e2e/lvmtest"
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/restclient"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	snapname     = "snaptest"
+	snapTestName string
+)
+
+// TestSnapshot creates a volume and snapshots, runs further tests on it and
+// finally deletes the volume
+func TestSnapshot(t *testing.T) {
+	var err error
+	var brickPaths []string
+	snapTestName = t.Name()
```

Comment:
you can use t.Name() directly, no need to assign it to a variable

Reply:
That is not possible, t.Name() will change the name, when executing sub tests